e2e-tests/support/user-tasks.js

```diff
@@ -44,6 +47,22 @@ module.exports = {
           },
         );
       },
+
+      async insertManyTfmDeals(numberOfDealsToInsert) {
```

Comment:
Could we add some documentation? Little hard to follow, fair amount of things going on in the for loop.

Reply:
Added docstring